package.json

```diff
@@ -67,6 +67,7 @@
     "react-dom": "^17.0.2",
     "react-hook-form": "^7.36.1",
     "react-hotkeys-hook": "^3.4.7",
+    "react-rangeslider": "^2.2.0",
```

Comment:
I don't think we should be using another library. We already have `Slider` component, please take a look at `src/components/dls/Slider/index.tsx`.